source/common/common/utility.h

```diff
@@ -96,21 +96,13 @@ class AccessLogDateTimeFormatter {
 };
 
 /**
- * Production implementation of SystemTimeSource that returns the current time.
+ * Real-time implementation of TimeSource.
```

Comment:
This makes it sounds like "real time OS", but that's not the implication here. I'm not sure what's a better phrase - maybe "Real clock implementation of TimeSource"? I think the type name is OK, but in the description it would be good to avoid "Real-time".

Reply:
Good point. I struggled with this for a few minutes, and found this:

https://whatis.techtarget.com/definition/wall-time-real-world-time-or-wall-clock-time

So I'm going to leave all the class names and files as is, but in the comments use the term "real-world time". I kind of like "wall-clock time" as well, but let's see how this fits.